platform/barefoot/sonic-platform-modules-imt/sonic_platform/sensors.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+
+'''
+data argument is in "sensors -A -u" format, example:
+coretemp-isa-0000
+Package id 0:
+  temp1_input: 37.000
+  temp1_max: 82.000
+  temp1_crit: 104.000
+  temp1_crit_alarm: 0.000
+Core 0:
+  temp2_input: 37.000
+  ...
+'''
+def _sensors_chip_parsed(data: str):
+	def kv(line):
+		k, v, *_ = [t.strip(': ') for t in line.split(':') if t] + ['']
+		return k, v
+	chip, *data = data.strip().split('\n')
+	chip = chip.strip(': ')
+	sensors = []
+	for line in data:
+		if not line.startswith(' '):
+			sensor_label = line.strip(': ')
+			sensors.append((sensor_label, {}))
+			continue
+		if len(sensors) == 0:
+			raise RuntimeError(f'invalid data to parse: {data}')
+		attr, value = kv(line)
+		sensor_label, sensor_data = sensors[-1]
+		sensor_data.update({attr: value})
+	return chip, dict(sensors)
+
+'''
+Example of returned dict:
+{
+	'coretemp-isa-0000': {
+		'Core 1': { "temp1_input": 40, ...  },
+		'Core 2': { ... }
+	}
+}
+'''
+def _sensors_get() -> dict:
+	data = subprocess.check_output("/usr/bin/sensors -A -u", shell=True, text=True)
+	data = data.split('\n\n')
+	data = [_sensors_chip_parsed(chip_data) for chip_data in data if chip_data]
+	data = dict(data)
+	return data
+
+def _value_get(d: dict, key_prefix, key_suffix=''):
```

Comment:
If I'm not mistaken it contradicts the PEP code style that we use function which name started with underscore outside the module

Reply:
fixed, also all files checked with pycodestyle and fixed all errors